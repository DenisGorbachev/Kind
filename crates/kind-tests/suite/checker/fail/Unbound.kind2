Tew : U60
Tew = owo