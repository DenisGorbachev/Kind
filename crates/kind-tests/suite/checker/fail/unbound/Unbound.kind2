type Test (a: Type) ~ (b: Type) {
    trick (x: a) (xs: List a)
}


type Test (b: Type) {
    trick (x: a)
}

