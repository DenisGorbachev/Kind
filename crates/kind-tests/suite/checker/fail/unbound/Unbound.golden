   ERROR  Defined multiple times for the same name

      /--[suite/checker/fail/unbound/Unbound.kind2:2:5]
      |
    1 |    type Test (a: Type) ~ (b: Type) {
    2 |        trick (x: a) (xs: List a)
      |        v----
      |        \The first ocorrence
    3 |    }
      :     
    7 |        trick (x: a)
      |        v----
      |        \Second occorrence here!

      Hint: Rename one of the definitions or remove and look at how names work in Kind at https://github.com/Kindelia/Kind2/blob/master/guide/naming.md

   ERROR  Defined multiple times for the same name

      /--[suite/checker/fail/unbound/Unbound.kind2:1:6]
      |
    1 |    type Test (a: Type) ~ (b: Type) {
      |         v---
      |         \The first ocorrence
    2 |        trick (x: a) (xs: List a)
    3 |    }
      :     
    6 |    type Test (b: Type) {
      |         v---
      |         \Second occorrence here!

      Hint: Rename one of the definitions or remove and look at how names work in Kind at https://github.com/Kindelia/Kind2/blob/master/guide/naming.md

